ephios/extra/templates/extra/widgets/previous_comments.html

```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+<div>
+    {% for comment in comments %}
+        <p class="mt-0 mb-0 d-flex">
```

Comment:
I think when displaying comments, the need some kind of indicator who can see them. When doing the disposition, I might think a user can see a comment I wrote 5 days ago, even though I might have checked the box to hide it from them :/